### PR TITLE
T332786: wmde13 security release for mw1.39.4

### PR DESCRIPTION
**Pull request by darthmon**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -4,13 +4,23 @@ This file is only intended to serve the developers of this repository and of the
...
-For a MediaWiki update to 1.39.2
+- [MediaWiki security release announcement](https://lists.wikimedia.org/hyperkitty/list/wikitech-l@lists.wikimedia.org/message/6UQBHI5FWLATD7QO7DI4YS54U7XSSLAN/)
+- [MediaWiki full release notes](https://www.mediawiki.org/wiki/Release_notes/1.39)
 
 | Suite Version | MediaWiki release | Date available | Run number | Release task  |
 |---------------|-------------------|----------------|------------| --------------|
-| wmde.12 ([spec](),[metadata]()) | 1.39.2    | March 2023 | ??? | [T332786](https://phabricator.wikimedia.org/T332786) |
+| wmde.13 ([spec](),[metadata]()) | MediaWiki (1.39.3, PHP 8.0, PHP 8.1, PHP 8.2) | April 2023 | ??? | [T332786](https://phabricator.wikimedia.org/T332786) |
+
+## Work In Progress - April: 2023: 1.38.6 Security release
```
Comment (addshore) [inline CHANGES.md:16]: In the past we used a single heading for a single set of security releases
So the heading for wmde.13 should be joined with wmde.12
Together these make up the April suite security release.

File: CHANGES.md [hunk 2]
```diff
@@ -4,13 +4,23 @@ This file is only intended to serve the developers of this repository and of the
...
+## Work In Progress - April: 2023: 1.39.3 Security release
 
-For a MediaWiki update to 1.39.2
+- [MediaWiki security release announcement](https://lists.wikimedia.org/hyperkitty/list/wikitech-l@lists.wikimedia.org/message/6UQBHI5FWLATD7QO7DI4YS54U7XSSLAN/)
+- [MediaWiki full release notes](https://www.mediawiki.org/wiki/Release_notes/1.39)
 
 | Suite Version | MediaWiki release | Date available | Run number | Release task  |
 |---------------|-------------------|----------------|------------| --------------|
-| wmde.12 ([spec](),[metadata]()) | 1.39.2    | March 2023 | ??? | [T332786](https://phabricator.wikimedia.org/T332786) |
+| wmde.13 ([spec](),[metadata]()) | MediaWiki (1.39.3, PHP 8.0, PHP 8.1, PHP 8.2) | April 2023 | ??? | [T332786](https://phabricator.wikimedia.org/T332786) |
```
Comment (addshore) [inline CHANGES.md:14]: As in https://github.com/wmde/wikibase-release-pipeline/pull/415 the PHP version here should bethe version actually in the container.

File: versions/wmde12.env [hunk 3]
```diff
@@ -1,34 +1,34 @@
 WMDE_RELEASE_VERSION=wmde.12
-WIKIBASE_BRANCH_NAME=REL1_39
-MEDIAWIKI_BRANCH_NAME=REL1_39
+WIKIBASE_BRANCH_NAME=REL1_38
```
Comment (addshore) [inline versions/wmde12.env:2]: This is all in https://github.com/wmde/wikibase-release-pipeline/pull/415
It looks like this PR should be targeting that branch, not master
Then we wouldn't be seeing this file change also in this PR
https://stackoverflow.com/questions/24159036/how-to-modify-a-pull-request-on-github-to-change-target-branch-to-merge-into

File: versions/wmde13.env [hunk 4]
```diff
@@ -0,0 +1,41 @@
+WMDE_RELEASE_VERSION=wmde.13
```
Comment (addshore) [inline versions/wmde13.env:1]: env file looks good to me

**Comment by addshore** [comment]
Re running failed jobs 
image

File: CHANGES.md (hunk 1 above)
Comment (darthmon) [reply CHANGES.md:16]: will join them then! :)

**Comment by addshore** [comment]
🎉

File: CHANGES.md (hunk 2 above)
Comment (darthmon) [reply CHANGES.md]: as per https://hub.docker.com/layers/library/php/8.1-apache/images/sha256-5ffb8acb1697f09cbad85693fbf6c16ed107c3123cfdd841652cb7f400f8c66f?context=explore we could add debian/apache2 2.4.57-2

Comment (darthmon) [reply CHANGES.md]: as per https://github.com/wikimedia/mediawiki-docker/blob/5dfcc29a3820d6129124f8eb154a6632d61a3162/1.39/apache/Dockerfile and https://hub.docker.com/layers/library/php/8.1-apache/images/sha256-5ffb8acb1697f09cbad85693fbf6c16ed107c3123cfdd841652cb7f400f8c66f?context=explore it should be PHP 8.1.20

File: versions/wmde12.env (hunk 3 above)
Comment (rti) [reply versions/wmde12.env]: resolved by later changes

**Comment by darthmon** [comment]
mw1.39.5 has caught up with us which makes this PR obsolete - the decision of closing this PR instead of reworking it is that the main branch has changed a lot since this PR was open.

**Pull request closed without merging**